packages/server/lib/util/require_async.ts

```diff
@@ -0,0 +1,78 @@
+import _ from 'lodash'
+import * as path from 'path'
+import * as cp from 'child_process'
+import * as inspector from 'inspector'
+import * as util from '../plugins/util'
+import * as errors from '../errors'
+import Debug from 'debug'
+
+const debug = Debug('cypress:server:require_async')
+
+let requireProcess: cp.ChildProcess | null
+
+interface RequireAsyncOptions{
```

Comment:
Nit: Linting - CI should be failing, if it's not, we should look into it. You should be able to configure VS Code to infer indentation based on the file's current indentation, though.

Reply:
Done in 87c1202ba409dcdae7ad3a0031177dc0eb7622cb